### PR TITLE
Fix for acra-configui auth_off mode

### PR DESCRIPTION
**Pull request by mozhmike**

### CODE REVIEW

File: cmd/acra_configui/acra_configui.go [hunk 1]
```diff
@@ -399,9 +402,14 @@ func main() {
 		os.Exit(1)
 	}
 
-	err = loadAuthData()
-	if err != nil {
-		os.Exit(1)
+	if *authMode == "auth_off" {
+		log.Info("HTTP Basic Auth is turned off")
```
Comment (vixentael) [inline cmd/acra_configui/acra_configui.go]: warning?

Comment (mozhmike) [reply cmd/acra_configui/acra_configui.go]: true

File: cmd/acra_configui/acra_configui.go [hunk 2]
```diff
@@ -399,9 +402,14 @@ func main() {
 		os.Exit(1)
```
Comment (Lagovas) [inline cmd/acra_configui/acra_configui.go]: print about error for user

File: cmd/acra_configui/acra_configui.go [hunk 3]
```diff
@@ -399,9 +402,14 @@ func main() {
...
-	err = loadAuthData()
-	if err != nil {
-		os.Exit(1)
+	if *authMode == "auth_off" {
+		log.Info("HTTP Basic Auth is turned off")
+	} else {
+		log.Infof("HTTP Basic Auth mode: %v", *authMode)
+		err = loadAuthData()
+		if err != nil {
+			os.Exit(1)
```
Comment (Lagovas) [inline cmd/acra_configui/acra_configui.go]: print something about error

**Pull request merged**